### PR TITLE
Update companies based on 2021 survey

### PR DESCRIPTION
**Pull request by CaroFG**

### CODE REVIEW

**Review by dichotommy — APPROVED** [review]
LGTM 😁

File: assets/companies.md [hunk 1]
```diff
@@ -2,17 +2,31 @@
 
 | Companies & Organizations| Description |How they use us |Source  |
 |--|--|--|--|
+|[asomas](https://asomas.ai/)|AI platform|Text search over database|2021 survey|
```
Comment (gmourier) [inline assets/companies.md:5]:
```suggestion
|[asomas](https://asomas.ai/)|AI platform|Text search|May 2021 Search Survey|
```

File: assets/companies.md [hunk 2]
```diff
@@ -2,17 +2,31 @@
...
 | [Aqua Security](https://avd.aquasec.com/) | Vulnerability database | 	Search by keyword or CVE-ID | [tweet](https://twitter.com/simarpreet7/status/1339365728894349312) |
+|[Bestseller](https://about.bestseller.com/)|B2C fashion|Searching on different internal IDs in their media storage system|2021 survey|
```
Comment (gmourier) [inline assets/companies.md:7]:
```suggestion
|[Bestseller](https://about.bestseller.com/)|B2C fashion|Internal Search|May 2021 Search Survey|
```

File: assets/companies.md [hunk 3]
```diff
@@ -2,17 +2,31 @@
...
 |[Beyond Code](https://beyondco.de/docs/expose/introduction)|Learning platform|Documentation search|[Marcel Pociot's website](https://pociot.dev/35-blazing-fast-search-with-meilisearch-and-laravel-forge)|
+|[Bonfire](https://bonfirenetworks.org/)|Social network||2021 survey|
```
Comment (gmourier) [inline assets/companies.md:9]:
```suggestion
|[Bonfire](https://bonfirenetworks.org/)|Social network||May 2021 Search Survey|
```

File: assets/companies.md [hunk 4]
```diff
@@ -2,17 +2,31 @@
...
 |[Caroom](https://www.caroom.fr/)|Car comparison website|Car search||
+|[coliquio](https://www.coliquio.de/)|Social network for medicine||2021 survey|
```
Comment (gmourier) [inline assets/companies.md:11]:
```suggestion
|[coliquio](https://www.coliquio.de/)|Social network for medicine||May 2021 Search Survey|
```

File: assets/companies.md [hunk 5]
```diff
@@ -2,17 +2,31 @@
...
 |[Docxtemplater](https://docxtemplater.com/)|Create templates and generate some Word, Powerpoint and Excel documents from javascript objects or JSON|Site search (modules, documentation, faq)||
+|[Empatix](https://www.empatix.no/)|Digital agency||2021 survey|
```
Comment (gmourier) [inline assets/companies.md:13]:
```suggestion
|[Empatix](https://www.empatix.no/)|Digital agency||May 2021 Search Survey|
```

File: assets/companies.md [hunk 6]
```diff
@@ -2,17 +2,31 @@
...
+|[every day](https://www.every-day.nl/)|Digital agency||2021 survey|
```
Comment (gmourier) [inline assets/companies.md:14]:
```suggestion
|[every day](https://www.every-day.nl/)|Digital agency||May 2021 Search Survey|
```

File: assets/companies.md [hunk 7]
```diff
@@ -2,17 +2,31 @@
...
 |[Hedira](https://www.hedira.io/)|Houseplant app that sends you tailored care reminders to your phone.|Plant search|[MeiliSearch Community Slack](https://meilicommunity.slack.com/archives/C01F44SSCCA/p1606344526015300)|
 |[Juice Brothers](https://juicebro.com/en/products/drinks-en/juices-en/)|Juice Store|MeiliSearch  based filters on the PLP|[tweet](https://twitter.com/markdevri_es/status/1347178895271989250)|
-| [Platform.sh](https://docs.platform.sh/) | Platform-as-a-Service built especially for continuous deployment |	Cross-site search  | [tweet](https://twitter.com/robertDouglass/status/1262395403363921922) |
+|[lamalama](https://lamalama.nl/)|Digital agency|E-commerce product search|2021 survey|
```
Comment (gmourier) [inline assets/companies.md:17]:
```suggestion
|[lamalama](https://lamalama.nl/)|Digital agency|E-commerce product search|May 2021 Search Survey|
```

File: assets/companies.md [hunk 8]
```diff
@@ -2,17 +2,31 @@
...
+|[mycoach](https://www.mycoachsport.com/)|Athlete Management System for professional teams| Search for players, training sessions, drills, and games|2021 survey|
```
Comment (gmourier) [inline assets/companies.md:18]:
```suggestion
|[mycoach](https://www.mycoachsport.com/)|Athlete Management System for professional teams| App Search |May 2021 Search Survey|
```

File: assets/companies.md [hunk 9]
```diff
@@ -2,17 +2,31 @@
...
+|[nobrinde](https://nobrinde.com/)|E-commerce|Product Search|2021 survey|
```
Comment (gmourier) [inline assets/companies.md:19]:
```suggestion
|[nobrinde](https://nobrinde.com/)|E-commerce|Product Search|May 2021 Search Survey|
```

File: assets/companies.md [hunk 10]
```diff
@@ -2,17 +2,31 @@
...
+|[Noppo](https://noppo.pro/)|Computer software company|E-commerce product search|2021 survey|
```
Comment (gmourier) [inline assets/companies.md:20]:
```suggestion
|[Noppo](https://noppo.pro/)|Computer software company|E-commerce product search|May 2021 Search Survey|
```

File: assets/companies.md [hunk 11]
```diff
@@ -2,17 +2,31 @@
...
+|[owow](https://owow.agency/)|Digital agency||2021 survey|
```
Comment (gmourier) [inline assets/companies.md:21]:
```suggestion
|[owow](https://owow.agency/)|Digital agency||May 2021 Search Survey|
```

File: assets/companies.md [hunk 12]
```diff
@@ -2,17 +2,31 @@
...
+|[Platform.sh](https://docs.platform.sh/) | Platform-as-a-Service built especially for continuous deployment |	Cross-site search  | [tweet](https://twitter.com/robertDouglass/status/1262395403363921922) |
 |[Quasar](https://next.quasar.dev/)|MIT licensed open-source Vue.js based framework| Site search| [MeiliSearch Community Slack](https://meilicommunity.slack.com/archives/C01F44SSCCA/p1614346362002000)|
 |[Ruby Toolbox](https://www.ruby-toolbox.com/)|Find actively maintained & popular open source software libraries for the Ruby programming language|Search for Ruby libraries|[official site](https://www.ruby-toolbox.com/blog/2021-03-19/search-speed-improvements)|
+|[showcase](https://getshowcase.io/)|Searchable Video SaaS|Searching through text transcripts|2021 survey|
```
Comment (gmourier) [inline assets/companies.md:25]:
```suggestion
|[showcase](https://getshowcase.io/)|Searchable Video SaaS|Text transcripts search|May 2021 Search Survey|
```

File: assets/companies.md [hunk 13]
```diff
@@ -2,17 +2,31 @@
...
 |[Tauri App](https://tauri.studio/en/)|Toolkit to make applications for the major desktop platforms|Documentation search|[tweet](https://twitter.com/TauriApps/status/1361195976946577413)|
-
-
-
-
+|[thatsup](https://thatsup.se/)|Swedish city guides||2021 survey|
```
Comment (gmourier) [inline assets/companies.md:27]:
```suggestion
|[thatsup](https://thatsup.se/)|Swedish city guides||May 2021 Search Survey|
```

File: assets/companies.md [hunk 14]
```diff
@@ -2,17 +2,31 @@
...
+|[Turkois](https://turkois.io/)|Digital agency||2021 survey|
```
Comment (gmourier) [inline assets/companies.md:28]:
```suggestion
|[Turkois](https://turkois.io/)|Digital agency||May 2021 Search Survey|
```

File: assets/companies.md [hunk 15]
```diff
@@ -2,17 +2,31 @@
...
+|[vembla](https://www.vembla.se/)| Food delivery|Search in the product catalog|2021 survey|
```
Comment (gmourier) [inline assets/companies.md:29]:
```suggestion
|[vembla](https://www.vembla.se/)| Food delivery|Product Catalog Search|May 2021 Search Survey|
```

File: assets/companies.md [hunk 16]
```diff
@@ -2,17 +2,31 @@
...
+|[Vylink](https://bike.vylink.com/)|E-bike company||2021 survey|
```
Comment (gmourier) [inline assets/companies.md:30]:
```suggestion
|[Vylink](https://bike.vylink.com/)|E-bike company||May 2021 Search Survey|
```

File: assets/companies.md [hunk 17]
```diff
@@ -2,17 +2,31 @@
...
+|[White Rabbit Digital Brands](https://www.whiterabbitdigital.uk/)|Digital agency|Use search in their dating CRM|2021 survey|
```
Comment (gmourier) [inline assets/companies.md:31]:
```suggestion
|[White Rabbit Digital Brands](https://www.whiterabbitdigital.uk/)|Digital agency|App Search|May 2021 Search Survey|
```

File: assets/companies.md [hunk 18]
```diff
@@ -2,17 +2,31 @@
...
+|[wux](https://wux.nl/)|Digital agency|Dynamic searching their ATS system, for searching in candidates, vacancies, companies, etc. Searching in their review website. Product display in their webshops.|2021 survey|
```
Comment (gmourier) [inline assets/companies.md:32]:
```suggestion
|[wux](https://wux.nl/)|Digital agency|App Search|May 2021 Search Survey|
```

**Review by gmourier — COMMENTED** [review]
@tpayet Can we share this information? The Search Survey did not strictly specify that the information could be shared since it was made to feedback analysis in a private way.

@CaroFG Following the Search Survey, the definition of `App Search` is "App Search represents the use-case of searching domain objects that are not products or text only content. (Ex: social events, doctors, cars to rent, e-learning courses, real estate etc..)."

**Pull request closed without merging**